triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -276,23 +282,24 @@ trait AbstractTriggerServiceTest
               Record(
                 None,
                 Seq(
-                  RecordField(value = Some(Value().withParty("Alice"))),
+                  RecordField(value = Some(Value().withParty(aliceAcs.unwrap))),
                   RecordField(value = Some(Value().withInt64(42)))))),
           ))
-        submitCmd(client, "Alice", cmd)
+        submitCmd(client, aliceAcs.unwrap, cmd)
       }
       // Query ACS until we see a B contract
-      // format: off
```

Comment:
As mentioned on slack, this was broken before. The solution isn’t particularly pretty but there is an assertion to guard against accidental party reuse so I would suggest to leave something like a proper per-test party allocation for a separate PR.